src/main/java/dk/cngroup/lentils/controller/ProgressController.java

```diff
@@ -63,10 +66,34 @@ public String listProgress(final Model model) {
 
     @GetMapping(value = "/teamsList")
     public String listTeams(final Model model) {
-        model.addAttribute("teams", teamService.getAll());
+        List<Team> teams = teamService.getAll();
+        List<Cypher> cyphers = cypherService.getAllCyphersOrderByStageAsc();
+        boolean allTeamsStarted = true;
+        for (Team team : teams) {
+            if (!statusService.isStatusInDbByCypherAndTeam(cyphers.get(0), team)) {
+                allTeamsStarted = false;
+            }
+        }
+        model.addAttribute("allTeamsStarted", allTeamsStarted);
+        model.addAttribute("teams", teams);
         return TEAM_LIST;
     }
 
+    @GetMapping(value = "/startGame")
+    public String initializeGameForAllTeams() {
+        List<Cypher> cyphers = cypherService.getAllCyphersOrderByStageAsc();
+        List<Team> teams = teamService.getAll();
+        for (Team t : teams) {
+            if (!statusService.isStatusInDbByCypherAndTeam(cyphers.get(0), t)) {
```

Comment:
tady by taky mohla byt metoda v servise, ktera nastavi pro dany tym status vsech sifer, protoze stejny kod se uz v aplikaci nachazi v klientske casti a tady je to jen zduplikovane.